lib/archethic_web/live/chains/node_shared_secrets_live.ex

```diff
@@ -0,0 +1,187 @@
+defmodule ArchethicWeb.NodeSharedSecretsChainLive do
+  @moduledoc false
+
+  use ArchethicWeb, :live_view
+
+  alias Archethic.{
+    TransactionChain,
+    TransactionChain.Transaction,
+    TransactionChain.TransactionData,
+    TransactionChain.TransactionData.Ownership,
+    PubSub,
+    SharedSecrets
+  }
+
+  alias ArchethicWeb.{ExplorerView, WebUtils}
+  alias Phoenix.{LiveView, View}
+
+  @display_limit 10
+  @txn_type :node_shared_secrets
+
+  @spec mount(_parameters :: map(), _session :: map(), socket :: LiveView.Socket.t()) ::
+          {:ok, LiveView.Socket.t()}
+  def mount(_parameters, _session, socket) do
+    if connected?(socket) do
+      PubSub.register_to_new_transaction_by_type(@txn_type)
+    end
+
+    tx_count = TransactionChain.count_transactions_by_type(@txn_type)
+
+    socket =
+      case SharedSecrets.genesis_address(@txn_type) do
+        nil ->
+          socket
+          |> assign(:tx_count, 0)
+          |> assign(:nb_pages, 0)
+          |> assign(:nb_authorized_nodes, 0)
+          |> assign(:current_page, 1)
+          |> assign(:transactions, [])
+
+        address when is_binary(address) ->
+          nb_authorized_nodes = nb_of_authorized_keys(address)
```

Comment:
Here you have to pass the last address, actually it show only 1 authorized node as it get the genesis address and no transaction exists for it

Reply:
really>?

Reply:
done